pystiche_papers/johnson_alahi_li_2016/core.py

```diff
@@ -135,9 +136,16 @@ def johnson_alahi_li_2016_stylization(
     transformer = transformer.to(device)
 
     if impl_params and preprocessor is None:
+        """
```

Comment:
Not sure whats with the multiline `str`. Whats the problem with multiple lines of comments?